src/auth/reference_modules/saml.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python3
+import io
+import json
+import os
+
+from onelogin.saml2.auth import OneLogin_Saml2_Auth
```

Comment:
Add `requirements.txt` under `src/auth/reference_modules/` + install it during memgraph install, update documentation, how users should install all deps required to run reference modules.

Reply:
Added